### PR TITLE
Quick update to offsite names turning red

### PR DESCRIPTION
**Pull request by Appleguy15**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -544,15 +544,15 @@ function subval_sort($a, $subkey, $result) {
                 $startTime = $ttStart->format('Y-m-d H:i:s');
             // These is for making the IF statment shorter
                 $statName = $latestdata['statusname'];
-                $GRtime = $currTime > $myReturn;
+                $GRtime = '$currTime > $myReturn';
         // Start IF statement for contextual coloring        
-        if ($currTime > $startTime && $statName == 'Not Checked In' || $GRtime && $statName == 'Offsite' || $GRtime && $statName == 'Field Trip' || $GRtime && $statName == 'Late' || $GRtime && $statName == 'Independent Study') {
+        if ($currTime > $startTime && $statName == 'Not Checked In' || $GRtime && $statName == 'Offsite' || $GRtime && $statName == 'Late') {
```
Comment (badtzmaru) [inline index.php]: does this actually work?

**Comment by Appleguy15** [comment]
@badtzmaru Yep. Just tested it.

**Comment by badtzmaru** [comment]
before we merge this can you make sure that this is not happening.
ampmfix

**Comment by badtzmaru** [comment]
i thought that we had fixed the am/pm thing

**Comment by Appleguy15** [comment]
@badtzmaru Idk me too. It might have happened becasue of connection.php

**Pull request merged**